From cb09a3d49742aa4b8e541334e2e49e8cb9b652e2 Mon Sep 17 00:00:00 2001
From: Marko Myllynen <myllynen@redhat.com>
Date: Sun, 12 Mar 2017 17:23:52 -0300
Subject: [PATCH] Fix send consolidation typo

Fix 60f9423b type for alpha kernel-features.h definition.

	* sysdeps/unix/sysv/linux/alpha/kernel-features.h
        (__ASSUME_RECV_SYSCALL): Replace duplicate by
        __ASSUME_SEND_SYSCALL.
---
 ChangeLog                                       | 6 ++++++
 sysdeps/unix/sysv/linux/alpha/kernel-features.h | 2 +-
 2 files changed, 7 insertions(+), 1 deletion(-)

diff --git a/ChangeLog b/ChangeLog
index e0acd7d..0aedae8 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,9 @@
+2016-03-12  Marko Myllynen  <myllynen@redhat.com>
+
+	* sysdeps/unix/sysv/linux/alpha/kernel-features.h
+	(__ASSUME_RECV_SYSCALL): Replace duplicate by
+	__ASSUME_SEND_SYSCALL.
+
 2017-03-10  Stefan Liebler  <stli@linux.vnet.ibm.com>
 
 	* math/auto-libm-test-out-catan: Regenerated.
diff --git a/sysdeps/unix/sysv/linux/alpha/kernel-features.h b/sysdeps/unix/sysv/linux/alpha/kernel-features.h
index 93ba14a..53f7611 100644
--- a/sysdeps/unix/sysv/linux/alpha/kernel-features.h
+++ b/sysdeps/unix/sysv/linux/alpha/kernel-features.h
@@ -33,6 +33,6 @@
 #define __NR_shmat __NR_osf_shmat
 
 #define __ASSUME_RECV_SYSCALL	1
-#define __ASSUME_RECV_SYSCALL	1
+#define __ASSUME_SEND_SYSCALL	1
 
 #endif /* _KERNEL_FEATURES_H */
-- 
2.9.3


